editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -107,17 +107,25 @@ void TileMapEditorTilesPlugin::_update_transform_buttons() {
 		return;
 	}
 
-	if (tile_set->get_tile_shape() == TileSet::TILE_SHAPE_SQUARE || selection_pattern->get_size() == Vector2i(1, 1)) {
-		transform_button_rotate_left->set_disabled(false);
-		transform_button_rotate_left->set_tooltip_text("");
-		transform_button_rotate_right->set_disabled(false);
-		transform_button_rotate_right->set_tooltip_text("");
+	if (!selection_pattern->is_empty() && Object::cast_to<TileSetScenesCollectionSource>(tile_set->get_source(selection_pattern->get_cell_source_id(Vector2i())).ptr())) {
```

Comment:
You need test all cells of the pattern to see if at least one is a scene, not simply the one in the `(0, 0)` position.

Reply:
For some reason I assumed you can't mix them .-.